clients/wallet/src/horizon.rs

```diff
@@ -131,6 +132,33 @@ where
 #[derive(Deserialize, Debug)]
 pub struct HorizonTransactionsResponse {
 	pub _embedded: EmbeddedTransactions,
+	_links: HorizonLinks,
+}
+
+#[allow(dead_code)]
+impl HorizonTransactionsResponse {
+	fn previous_page(&self) -> String {
+		self._links.prev.href.clone()
+	}
+
+	pub(crate) fn next_page(&self) -> String {
```

Comment:
Somehow I would assume this function to return the new `HrefPage` struct instead of a string.